Integrate WG and TF information

iadawn opened the pull request.

daniel-montalvo committed:
Conformance testing work statement 2024

iadawn committed:
Working on AGWG

iadawn committed:
Update for AG materials

daniel-montalvo committed:
Adds aria info draft

daniel-montalvo committed:
More Consistency for acronyms

daniel-montalvo committed:
Update _about/groups/agwg/task-forces/conformance-testing/work-statement.md

Co-authored-by: Kathy Eng <kengdoj@users.noreply.github.com>

daniel-montalvo committed:
Merge pull request #244 from w3c/AGWG-task-forces-info

Conformance testing work statement 2024

iadawn committed:
All WG pages

daniel-montalvo committed:
Absolute URIs for wikis

daniel-montalvo committed:
Change nav order

daniel-montalvo committed:
WCAG2ICT info

daniel-montalvo committed:
Add missing aria info (mainly for review)

daniel-montalvo committed:
Updates ARIA info

ruoxiran committed:
Update personalization to Adapt

ruoxiran committed:
Update communication.md

ruoxiran committed:
Update contribute.md

ruoxiran committed:
Update contribute.md

ruoxiran committed:
Update participate.md

ruoxiran committed:
Update apawg.md

ruoxiran committed:
Update planning.md

iadawn committed:
Update TFs

iadawn committed:
Merge pull request #264 from w3c/master

Update from master

remibetin commented on _about/groups/task-forces/coga.md:
```diff
@@ -0,0 +1,112 @@
+---
+title: Cognitive and Learning Disabilities Accessibility Task Force (COGA)
+ref: /about/groups/task-forces/coga/
```
```suggestion
ref: /about/groups/task-forces/coga/
permalink: /about/groups/task-forces/coga/
```
@iadawn This page is missing a `permalink` attribute in the frontmatter, causing the build to fail.

iadawn committed:
Add missing pages and some more updates

iadawn committed:
Fix task force link

shawna-slh commented:
I'm assuming I don't need to look at this now? Let me know if I do need to review, and specifically what I should look at.

(I will be involved with the content redesign in the hopefully near future.)

I'm happy to let @iadawn and @remibetin handle this when it's ready for publication. :-)

kfranqueiro reviewed (commented):
I spent some time looking through this, found a few things (I stopped myself at some point to not consume too much time), and have a couple of questions:

I know you mentioned in the email that navigation won't work as expected; does this extend to the absence of the sidebar and full breadcrumbs in group sub-pages and the sidebar still linking to `taskforces` without a hyphen?

I may be missing context, but is there a reason that `conformance-testing` and `wcag2ict` appear spread between `agwg/task-forces` and top-level `task-forces`? (`conformance-testing` even has 2 `work-statement.md`s, one in each.) It seems like the top-level ones are what typically get linked right now in this repo, but looking at the existing site, it looks like many are indeed placed under relevant subfolders; are we expecting to consistently operate one way or the other here?

daniel-montalvo commented:
> I may be missing context, but is there a reason that `conformance-testing` and `wcag2ict` appear spread between `agwg/task-forces` and top-level `task-forces`? (`conformance-testing` even has 2 `work-statement.md`s, one in each.) It seems like the top-level ones are what typically get linked right now in this repo, but looking at the existing site, it looks like many are indeed placed under relevant subfolders; are we expecting to consistently operate one way or the other here?

Thanks much for pointing this out, @kfranqueiro . I am not sure what structure we want to follow here. Seems we want to change to top-level (/about/groups/task-forces/) instead of using /about/groups/<xx>wg/task-forces/. If that is the case @iadawn let me know so that I can update these.

daniel-montalvo commented on _about/groups/task-forces/wcag2ict.md:
```diff
@@ -0,0 +1,99 @@
+
+### Work Statement
+
+The [WCAG2ICT Task Force Work Statement](https://www.w3.org/WAI/GL/task-forces/wcag2ict/work-statement) defines the objectives, scope, approach, and participation of the Task Force. The work statement has been approved by the Accessibility Guidelines WG.
```
Thanks for pointing this out @kfranqueiro Yes,we should definitely update this. Just didn't do that because I am unclear in terms of structure (as said above).

iadawn committed:
Fix WG links

iadawn commented on _about/groups/task-forces.md:
```diff
@@ -24,33 +24,33 @@ ones, are listed on each Working Group's home page.
     states, and properties.    
 
-**[Cognitive and Learning Disabilities Accessibility Task Force](https://www.w3.org/WAI/GL/task-forces/coga/)** (Cognitive A11Y TF) under the [APA Working Group](/about/groups/apa/) and the [AG Working Group](/about/groups/agwg/)
+**[Cognitive and Learning Disabilities Accessibility Task Force](/about/groups/task-forces/coga/)** (Cognitive A11Y TF) under the [APA Working Group](/about/groups/apa/) and the [AG Working Group](/about/groups/agwg/)
```
Fixed

iadawn commented:
Obviously I was in two minds regarding the positioning of the task force additional material. I think I am settling on having it in the groups/task-forces/ section rather than associated with a specific WG. This makes it easier to deal with TFs in multiple WGs.

I will sort out the AG stuff

daniel-montalvo committed:
new TF structure

daniel-montalvo committed:
Fix links

daniel-montalvo committed:
Add missing "-"

daniel-montalvo commented on _about/groups/task-forces/wcag2ict.md:
```diff
@@ -0,0 +1,99 @@
+
+- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/55145/);
+- [Teleconferences of the task force](https://www.w3.org/WAI/GL/task-forces/wcag2ict/#teleconferences) (also see [meeting minutes](https://www.w3.org/WAI/GL/task-forces/wcag2ict/minutes));
+- Face to face meetings;
```
i think we should keep link to minutes as-is for now until we decide what to do with the minutes tool.

daniel-montalvo committed:
Updates WCAG2ICT Overview page and links

daniel-montalvo committed:
Make IRC info clearer

daniel-montalvo commented on _about/groups/ariawg.md:
```diff
@@ -4,5 +4,94 @@ nav_title: '<span>ARIA: Accessible Rich Internet Applications <abbr title="Worki
+The ARIA WG conducts its work using a variety of synchronous and asynchronous tools. The [communication](aria/communication) page provides details about:
+
+- Teleconferences of the Working Group and its task forces (also see [meeting minutes](aria/minutes));
+- Face to face meetings (also see face to face [meeting pages](https://www.w3.org/WAI/ARIA/wiki/Meetings));
```
Again, I think let's keep this link as-is until we decide what to do with the minutes tool.;zzz

daniel-montalvo commented on _about/groups/ariawg.md:
```diff
@@ -4,5 +4,94 @@ nav_title: '<span>ARIA: Accessible Rich Internet Applications <abbr title="Worki
+- [Wiki](https://www.w3.org/WAI/ARIA/wiki/);
+- [Web-Based Surveys (WBS)](/2002/09/wbs/83726/);
+- [Issue Tracker](track/);
+- [ARIA source repository issue tracker](https://github.com/w3c/aria/issues).
```
GitHub is were the group tracks issues now.

daniel-montalvo committed:
Update ARIA page

daniel-montalvo commented:
Just realized our proposal and the W3C URI structure are slightly different.

- This proposal puts "wg" at the end of each working group acronym, for example, /about/groups/agwg/, whereas W3C structure puts "wg" before the group name, for instance, /groups/wg/ag
- This proposal spells out "task-forces" whereas W3C uses "tf'. For example, /about/groups/task-forces/wcag2ict/ versus /groups/tf/wcag2ict

daniel-montalvo committed:
Fix TOC

daniel-montalvo committed:
Remove duplicate TOC

daniel-montalvo committed:
Try summary box before TOC

daniel-montalvo committed:
Tweaks to contribute wording

iadawn committed:
Correct date

iadawn commented on _about/groups/ariawg/decision-policy.md:
```diff
@@ -0,0 +1,45 @@
+---
+title: ARIA Decision Policy
```
Given that it isn't archived yet. We will transition to Github and then archive.

iadawn commented:
@shawna-slh This is now ready for final checks and publishing. What will need to be done after transition is:

* Review for consistency across the groups
* Tidy up and simplify content - too many links around the place and too much repitition
* Review and resolve approach to the minutes tool - it is a good presentation but does mean that we need to keep a database and PHP files just for that
* Archive various decision policies

I will also need to sort out redirects. My preference is rather than do URI redirects to include an archived message on the pages and redirect from there. This can be done in one place (I think): _header.phi most probably. This could either be a static redirect from all of the old pages to https://w3.org/about/groups/ or something a bit more involved that redirects to the correct page. My preference and suggestion would be the former: much simpler and more future proof if some of those pages change.

Finally, will need an updated [navigation.xml](https://github.com/w3c/wai-website-data/blob/master/navigation.yml) to handle the new structure. I have the start of that but it will need some final tidying up and checking.

shawna-slh commented:
My understanding was this was a temporary fix and that the new content specialist would address the higher-level content restructuring, e.g., **https://lists.w3.org/Archives/Public/public-wai-cc/2021Jun/0022.html

(soon, which realistically is July depending on how we prioritize things)

shawna-slh commented:
> I'm happy to let @iadawn and @remibetin handle this when it's ready for publication. :-)

Handing to @remibetin for publication. Let me know if any questions or suggestions for me.

shawna-slh commented:
Hi @iadawn I think best not to add previous and next to these pages. I'm happy to discuss as useful.

iadawn committed:
Fix links and remove links to issue tracker

remibetin committed:
Fix link to ARIAWG Deliverables

remibetin committed:
Fix links

remibetin committed:
Test navigation

remibetin commented:
@iadawn The navigation is now displayed in the preview + I have fixed some broken links.

FYI, some linked content have not been moved:
- Maturity Model Task Force: https://www.w3.org/WAI/APA/task-forces/maturity-model/ (if not moved, will create an error in the "Task Forces" page)
- APG work statement: https://www.w3.org/WAI/ARIA/task-forces/practices/work-statement
- RQTF Work Statement : https://www.w3.org/WAI/APA/task-forces/research-questions/work-statement
- Silver Task Force Work Statement: https://www.w3.org/WAI/GL/task-forces/silver/work-statement
- Spoken Presentation Task Force Work Statement: https://www.w3.org/WAI/APA/task-forces/pronunciation/work-statement
- WAI-Adapt Task Force Work Statement: https://www.w3.org/WAI/APA/task-forces/adapt/work-statement

shawna-slh commented:
@ruoxiran & @iadawn I think:
* https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/task-forces/ should be deleted and any pointers should go to https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/task-forces/
* FAST and other task forces should be added to https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/task-forces/

remibetin committed:
Remove paginaton

remibetin commented:
> Hi @iadawn I think best not to add previous and next to these pages. 

@iadawn As discussed, I have updated the PR to remove the prev/next links.

iadawn commented:
> * https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/task-forces/ should be deleted and any pointers should go to https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/task-forces/

Already done

> * FAST and other active task forces should be added to https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/task-forces/

Agreed that this needs to be done. However, the content is not in there at the moment. Given the first goal is to get this material into Github as is, then updating the content to be a better reflection of reality can wait.

> * Maturity Model Task Force: https://www.w3.org/WAI/APA/task-forces/maturity-model/ (if not moved, will create an error in the "Task Forces" page)

Moved

> * APG work statement: https://www.w3.org/WAI/ARIA/task-forces/practices/work-statement

Done. Also added work statements from APG and CSS A11y

> * RQTF Work Statement : https://www.w3.org/WAI/APA/task-forces/research-questions/work-statement

Done.

> * Silver Task Force Work Statement: https://www.w3.org/WAI/GL/task-forces/silver/work-statement

Done.

iadawn committed:
Lots of clean up

All related to https://github.com/w3c/wai-about-wai/pull/245#issuecomment-2152298099

remibetin reviewed (commented):
@iadawn I have reviewed and added a couple comments.

> https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/task-forces/ should be deleted and any pointers should go to https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/task-forces/

> Already done

[APA Task Forces page](https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/task-forces/) still exists and [APA WG Page](https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/#task-forces) still points to it. Is this on purpose?

remibetin commented:
@iadawn What do you think of the navigation?

Feel free to directly edit the related PR: https://github.com/w3c/wai-website-data/pull/151

remibetin commented on _about/groups/task-forces/wcag2x-backlog.md:
```diff
@@ -0,0 +1,67 @@
+---
+title: WCAG 2.x Backlog Task Force (MATF) of the AG WG
```
```suggestion
title: WCAG 2.x Backlog Task Force of the AG WG
```

remibetin committed:
Fix links

iadawn committed:
Fix cut and paste error

Co-authored-by: Rémi Bétin <git@remibetin.fr>

iadawn commented on _about/groups/ariawg/work-statement.md:
```diff
@@ -0,0 +1,80 @@
+---
+title: ARIA Authoring Practices Task Force Work Statement
+permalink: /about/groups/ariawg/work-statement/
```
Hmm... the work-statement shouldn't be there. Removed

iadawn committed:
Remove replicated work statement

iadawn committed:
Add pronunciation work statement

iadawn commented on _about/groups/task-forces/pronunciation.md:
```diff
@@ -97,25 +97,21 @@ The Spoken Presentation Task Force uses the public-pronunciation@w3.org mailing
-
-**
\ No newline at end of file
+The [Spoken Presentation Accessibility Task Force Work Statement](/about/groups/task-forces/pronunciation/work-statement/) defines the initial objective, scope, approach, and participation of the Task Force.
```
Added

iadawn commented:
> [APA Task Forces page](https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/task-forces/) still exists and [APA WG Page](https://deploy-preview-245--wai-about-wai.netlify.app/about/groups/apawg/#task-forces) still points to it. Is this on purpose?

Yes. I think I mis read the first comment. There should be a page for each WG with a list of associated TFs. However, TF information is all managed in the /task-forces/ directory not within the WG directory structure.

remibetin committed:
Fix a link

remibetin committed:
Reset Gemfile.lock

remibetin committed:
Remove test navigation
